ethcore/src/miner/miner.rs

```diff
@@ -216,8 +214,6 @@ impl Miner {
 	pub fn new(options: MinerOptions, gas_pricer: GasPricer, spec: &Spec, accounts: Option<Arc<AccountProvider>>) -> Arc<Miner> {
```

Comment:
future work: change this constructor to accept solely an `Arc<Engine>` instead of `&Spec`.